dashboard/client/src/pages/node/NodeRow.tsx

```diff
@@ -195,28 +198,33 @@ export const WorkerRow = ({ node, worker }: WorkerRowProps) => {
       </TableCell>
       <TableCell align="center">{pid}</TableCell>
       <TableCell>
+        {/* {state !== "DEAD" && ( */}
```

Comment:
remove

Reply:
Yes, we should not leave unnecessary comment in the code. I will make the changes after getting the UI decision from @scottsun94 to decide whether to show the logs of dead worker/nodes